buildSrc/src/main/groovy/Classpaths.groovy

```diff
@@ -152,32 +148,32 @@ class Classpaths {
         }
     }
 
-    static void inheritJavaParser(Project p, String name = JAVA_PARSER_NAME) {
-        Configuration compile = compile p
-        addDependency compile, JAVA_PARSER_GROUP, name, JAVA_PARSER_VERSION
+    static void inheritJavaParser(Project p, String configName = JavaPlugin.IMPLEMENTATION_CONFIGURATION_NAME) {
```

Comment:
Is having a default `configName` on these methods actually beneficial? I would think it's better to be explicit in the caller, unless we never use anything but the default.

Reply:
done

Reply:
`inheritCommonsText` still has a default.

Reply:
And `inhertJavaxAnnotatiions`.